### ISSUE TITLE
Using a pre-filtered OTU table based on occurrence and relative abundance in spieceasi 

### ISSUE THREAD

**Issue by suahin** [zdk123/SpiecEasi#261]
According to the paper (PLoS Comput Biol, 2015), input table must be non-normalized raw OTU table,

I have question about that whether I use a pre-filtered OTU table based on occurrence and relative abundance.

ALso, can I merge two different OTU tables and run single spieceasi instead of multiple spieceasi ?

**Comment by zdk123** [comment]
Yes, you can pass in pre-filtered count data sets.. for the multiple OTU
tables, were they created from different sequencing runs? If so, you should
use the multi.spiec.easi method since these are technically two separate
compositional datasets.

**Comment by suahin** [comment]
Thank you for your reply :) I have one more question,

I found that there are some difference between multiple.spieceasi and single,spieceasi. 
I performed multiple and single spieceeasi using two OTU table and compared results
by substracting between OTU table interaction to remain only within table interaction. 
Total interaction of multiple spieceasi - between table interaction) vs. Total interaction of single spieceasi, like this.
Then, much more OTU interactions are found that are not exists in single.spieceasi.

So, I am curious about why this phenomenon occurs and wondered what is the exact results?
